src/System.Private.Interop/src/Interop/Interop.String.Unix.cs

```diff
@@ -11,24 +11,25 @@ namespace System.Runtime.InteropServices
     public partial class ExternalInterop
     {
 
-        internal static unsafe uint SysStringLen(void* pBSTR)
+        public static unsafe uint SysStringLen(void* pBSTR)
```

Comment:
This change is also incorrect.  There is something wrong with Mcg generated code or similar. This is same issue as BSTR.

Reply:
Can you fix it up, this was done since this function is being referenced in code generated by MCG.

Reply:
Yes, I will do it.